### PR TITLE
We require go 1.17 to build `crust`

### PR DESCRIPTION
**Pull request by wojtek-coreum**
This change is [Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/136)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Everyone is encouraged to run a chain locally for development and testing purpos
 
 Entire process of running local chain is automated by our tooling. The only prerequisites are:
 - `docker` and `tmux` installed from your favorite package manager
-- `go 1.16` or newer installed and available in your `PATH`
+- `go 1.17` or newer installed and available in your `PATH`
```
Comment (dzmitryhil) [inline README.md:17]: The version in the root go.mod is 1.18 but in the build 1.17, would it make sense to keep just one version?

**Review by wojtek-coreum — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/136)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @begmaroman, @dzmitryhil, @miladz68, @oliverasistemas, @xlab, and @ysv)

___
*[`README.md` line 17 at r1](https://reviewable.io/reviews/coreumfoundation/coreum/136#-N9_pN_tgte6Ty-TFqQI-r1-17:-N9aFApR8tXKC-VKUtOz:bicnpuv) ([raw file](https://github.com/coreumfoundation/coreum/blob/d982970601fbe0f600c0312f237df6963282698f/README.md#L17)):*
<details><summary><i>Previously, dzmitryhil (Dzmitry Hil) wrote…</i></summary><blockquote>

The version in the root go.mod is 1.18 but in the build 1.17, would it make sense to keep just one version?
</blockquote></details>

`crust` must be built using the go version available in OS by default. Then `crust` downloads the go version of our choice (`build/tools/tools.go`). CI on github still provides only go 1.17 so that module must support that version.

**Review by ysv — APPROVED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/136)* status: all files reviewed, 1 unresolved discussion (waiting on @begmaroman, @dzmitryhil, @miladz68, @oliverasistemas, and @xlab)

**Review by wojtek-coreum — COMMENTED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/136)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @begmaroman, @miladz68, @oliverasistemas, and @xlab)

**Pull request merged**